app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Parent exception for all vaccine-related issues."""
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """Raised when a visitor is not vaccinated."""
+    pass

```

Comment:
The `NotVaccinatedError` class should have a descriptive error message. For example: `NotVaccinatedError('The user is not vaccinated.')`